main.cpp

```diff
@@ -122,12 +147,44 @@ struct Computer
     //input the graphics accelerator to update drivers for
     //returns true if driver updated successfully;
 
+    double analyzeEnergyConsumption(int numberOfSecondsPoweredOn)
+    {
+        double consumed = numberOfSecondsPoweredOn * memoryInGB * .0345;
+        if(graphicsAccelerator.isGSyncCapable)
+        {
+            consumed *= 4;
+        }
+        std::cout << "for the sake of science; energy consumed: " << consumed << std::endl;
+        return consumed;
+
+    }
+    std::string memoryTopologyBlocksPerCore()
+    {
+        //number of cores & number of GB RAM
+        auto memoryPerCore = memoryInGB / static_cast<double>(numberOfProcessorCores);
```

Comment:
remove this cast
casting is covered in Project 4 Part 3.
Change the type of `numberOfProcessorCores` to eliminate the warning
- [ ] revise